themes/frontierline/sass/components/content-views/_content-grid.scss

```diff
@@ -0,0 +1,21 @@
+// Used for a custom static layout of content
+
+.grid-item {
+  box-sizing: border-box;
+  padding-left: 15px;
+  padding-right: 15px;
+  height: 360px;
+
+  @media #{$mq-phone-wide} {
+    float: left;
+    width: 50%;
```

Comment:
I missed this when looking for a grid. Let me take a look. I'd prefer to use it and scrap what I wrote if it works.